src/components/menuAppBar/MenuAppBar.tsx

```diff
@@ -24,9 +23,13 @@ const MenuAppBar = () => {
 
   return (
```

Comment:
A lot of conditional rendering inside of return. Imho this could be done in different way.
For example exclude component like navButton with isActive and activeIcon and inActiveIcon props. 
Also current page should be checked only once - not for every possible route.